### DISCUSSION TITLE
We want to make the USWDS site better for you

### DISCUSSION THREAD

**Discussion by jaclinec** [uswds/uswds#5151] [General]
We are currently looking for ways to improve the organization and navigation of the USWDS site. As a step towards that goal, we're hoping to discover what the most important tasks are that our users need to do on the US Web Design System site. What are the main things you are looking to do? What are the things you're trying to understand? 

Please tell us by taking a [5 minute survey](https://touchpoints.app.cloud.gov/touchpoints/0a9a9227/submit) so we can make things better for you on the site. 

The more responses we get, the better - it'll be open for about 3 weeks. Thank you!

**Comment by jaclinec** [comment]
Last call to have your voice heard! This [5 minute survey](https://touchpoints.app.cloud.gov/touchpoints/0a9a9227/submit) closes this Friday, March 3rd. Thank you!

**Comment by ryparker** [comment]
The survey closed on March 3, 2023, so I'm closing this invitation. The feedback will remain here. For current website feedback, please use [USWDS site issues](https://github.com/uswds/uswds-site/issues/new/choose).
